mne/report.py

```diff
@@ -913,6 +919,11 @@ class Report(object):
         the data. Defaults to ``False``.
 
         .. versionadded:: 0.21
+    auto_close : bool
```

Comment:
I wouldn't have made this a class-level attribute but rather a method-level `kwarg`. You really only need it for `add_figs_to_section` and `add_slider_to_section` or whatever where you're passing in your own figures, and that's really when you want control over having them be closed. It seems like if you add it as a method kwarg instead, then there should be a way to get CircleCI to produce what you actually want it to.